files/mender/mender.service

```diff
@@ -0,0 +1,15 @@
+[Unit]                     
+Description=Mender OTA update service                         
+After=systemd-resolved.service                                
+
+[Service]                  
+Type=idle                  
+User=root                  
+Group=root                 
+ExecStartPre=/bin/mkdir -p -m 0700 /data/mender               
+ExecStartPre=/bin/ln -sf /etc/mender/tenant.conf /var/lib/mender/authtentoken                                                        
```

Comment:
Hi @mirzak 
Can you tell us if this line is necessary?
I can see a note in this article "Editor: the tenant.conf is only needed if you are using the Mender hosted service"
https://mender.io/blog/notes-from-a-user-on-mender-without-yocto
But I cannot find any reference to tenant.conf in https://docs.mender.io/1.6